Dockerfile

```diff
@@ -227,7 +227,7 @@ COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/includ
      /opt/tensorrtserver/include/torch
 COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libtorch.so \
       /opt/tensorrtserver/lib/
-COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libthnvrtc.so \
+COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libcaffe2_nvrtc.so \
```

Comment:
Note that these changes and the pytorch changes are on the r19.08 branch. They will move into the master branch when NGC releases 19.08. So when we rebase r19.09 to master we will have them automatically. I guess normally it is better to wait until after NGC release and all the steps there before doing this branch.

Reply:
If you want, I can wait and then redo this branch.